xla/service/gpu/cudnn_fusion_compiler.cc

```diff
@@ -311,6 +311,34 @@ class GemmDimensionAdapter {
   const HloDotInstruction& dot_;
 };
 
+std::optional<std::shared_ptr<graph::Tensor_attributes>>
+HandleConstantHloToCudnnGraph(const HloInstruction& hlo, graph::Graph& graph) {
+  CHECK(hlo.IsConstant()) << "HLO is not a constant: " << hlo.ToShortString();
+  if (!ShapeUtil::IsScalar(hlo.shape())) {
+    VLOG(3) << "Currently only support fusing scalar in the graph";
+    return std::nullopt;
+  }
+  PrimitiveType constant_type = hlo.shape().element_type();
+  switch (constant_type) {
+    case BF16:
+      return graph.tensor(__nv_bfloat16(
+          hlo.literal()
+              .data<primitive_util::PrimitiveTypeToNative<BF16>::type>()[0]));
```

Comment:
Nit:

1. Could we avoid data duplication? Both `hlo.literal` and `graph.tensor` calls should not be duplicated.
2. `literal.GetFirstElement<NativeType>()` seems more suitable

Reply:
Done.